matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
```

Comment:
"Normal mode" is the state-less interface (where you provide every time the  state to the functions) while "optimized mode" is the stateful interface (where you provide the state once with wbm_setWorldFrame/wbm_updateState and then you use that one), right? 

If you always use the stateless/normal mode interface, I think you can remove both `wbm_setWorldFrame` and `wbm_updateState`, right?